packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/ShareDiagramDialog.tsx

```diff
@@ -23,8 +23,8 @@ export const ShareDiagramDialog = ({ onClose }: ShareDiagramDialogProps) => {
       var range = document.createRange();
       range.selectNodeContents(node);
       var selection = window.getSelection();
-      selection.removeAllRanges();
-      selection.addRange(range);
+      selection?.removeAllRanges();
+      selection?.addRange(range);
```

Comment:
Prefer a test of selection to invoke methods like:

```
if (selection) {
  ...
}
```

Reply:
Done